### DISCUSSION TITLE
Don't change edited default skins when there's a new default skin update

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#25438] [Ideas] [answered]
Would be nice if when you edit a default skin, it doesn't get reset to an updated version.
I use argon pro but I changed some ui elements around, and added some from the list, but when I updated osu it all got reset because of the new argon ui update.

Maybe there could be like a popup that asks you if you wanna reset next time you try to edit the skin or something?

**Comment by peppy** [comment]
It shouldn't change. Can you better explain what changed?

**Reply by bytenekos** [reply]
It basically replaced the entire skin ui to default, as if like I selected the default argon pro skin.

**Reply by peppy** [reply]
Please attach your skin or something.

**Reply by bytenekos** [reply]
Before update
![image](https://github.com/ppy/osu/assets/66184829/eaa4ee09-229a-42a5-8b81-05dcf1dc2d19)

After update
![image](https://github.com/ppy/osu/assets/66184829/eda59199-40d4-4f6a-9bce-4d2f444c7843)

**Reply by bytenekos** [reply]
I did not change the selected skin, using the exact same skin selection in the settings menu

**Reply by bytenekos** [reply]
https://github.com/ppy/osu/assets/66184829/9ea4ab23-44cb-40d6-8347-fcdbea4bfc15

Replicated

**Reply by peppy** [reply, marked as answer]
Tracking at https://github.com/ppy/osu/issues/25437

**Reply by PetHedgehog** [reply]
hello, im completely green here, i was just told i might get some answers here, this is the issue for me as well, concerning the changes i would like to ask if we could possibly get an option to use the pre changes argon skin or at least a skin file to add to laser as a standalone skin, this was heart dropping for me and im sure many others, skins being just visual shouldnt mean we should be forced to update, for some this might even make the game unbearable. idk what tracking means here as i dont know what im doing here at all but does it mean the issue is just considered or will it get fixed and we will get a solution of some sort?

**Reply by bdach** [reply]
@PetHedgehog please update to latest release

**Reply by PetHedgehog** [reply]
@bdach it didnt get fixed, still the same

**Reply by PetHedgehog** [reply]
idk why, might be cause esc autosaves ant it messed itself up but i really cant look at these new numbers and the style of the whole skin, i dont think the numbers were default style, im not sure anymore im panicking too much to keep my head cool

**Reply by bdach** [reply]
@PetHedgehog please attach your skin, then.

**Reply by PetHedgehog** [reply]
@bdach how? the file after export?

**Reply by bdach** [reply]
yes

**Reply by PetHedgehog** [reply]
@bdach 
[osu_ _argon_ pro (2022) (modified) (Unknown).zip](https://github.com/ppy/osu/files/13350946/osu_._argon_.pro.2022.modified.Unknown.zip)

**Reply by bdach** [reply]
Sorry but unfortunately that looks like your skin was overwritten and it cannot be recovered now. We do not generally want it to happen, it was an unfortunate occurrence caused by a bug in the implementation of the newest changes.

**Reply by PetHedgehog** [reply]
@bdach but can i make it back how it was? do the old style numbers exist? i seriously know nothing about how this works, they seem like the default ones are similar

**Reply by PetHedgehog** [reply]
@bdach the healthbar seems broken now after the update
![image](https://github.com/ppy/osu/assets/150826274/4b7bdc17-fafd-4675-a758-d3a77ba47b2a)

**Reply by peppy** [reply]
The old styles still exist. You can look for the "DefaultXXX" implementations in the skin editor.

> @bdach the healthbar seems broken now after the update ![image](https://user-images.githubusercontent.com/150826274/282839176-4b7bdc17-fafd-4675-a758-d3a77ba47b2a.png)

Update to the latest hotfix release, this is fixed.

**Reply by bdach** [reply]
Yes they are the default ones.

![image](https://github.com/ppy/osu/assets/20418176/b63923f7-c878-400c-b58e-f0e86ca47609)

**Reply by PetHedgehog** [reply]
running .1 hotfix says up to date

**Reply by PetHedgehog** [reply]
ok i managed to get my layout back but the healthbar had to be resized from a save copy i had, when adding a new one its inverted like in the screen shot, lazer is up to date
